convert/jenkinsjson/json/notification.go

```diff
@@ -0,0 +1,51 @@
+package json
+
+import (
+	"fmt"
+
+	harness "github.com/drone/spec/dist/go"
+)
+
+// ConvertNotification creates a Harness step for Notification plugin.
+func ConvertNotification(node Node, parameterMap map[string]interface{}) *harness.Step {
+
+	// Extract values from parameterMap
+	url, urlOk := parameterMap["endpoint"].(string)
+	urls := []string{}
+
+	if urlOk {
+		urls = append(urls, url)
+	} else {
+		urls = append(urls, "<+input>")
+	}
+
+	phase, _ := parameterMap["phase"].(string)
+	notes, _ := parameterMap["notes"].(string)
+
+	// Create the template
+	template := fmt.Sprintf(`{
+  		"phase": "%s",
```

Comment:
```suggestion
  		"status": "%s",
```

whenever the phase is completed in jenkins trace, then the value of status should be successful - if it's not, then leave this empty